evap/evaluation/templates/evap_evaluation_edit_js.html

```diff
@@ -1,32 +1,32 @@
 {% load static %}
 
 {% if editable %}
-    <script src="{% static 'js/sortable_form.js' %}"></script>
-    <script type="text/javascript">
-        rowChanged = function(row) {
-            name = $(row.find("select[id$=-contributor]")).find(":selected").text();
-            nameChanged = name && name != "---------";
-            idChanged = row.find("input[id$=-id]").val();
+    <script type="module">
+        import { makeFormSortable } from "{% static 'js/sortable-form.js' %}";
+        const rowChanged = function(row) {
+            const name = row.querySelector("select[id$=-contributor] > [selected]")?.textContent;
+            const nameChanged = name && name != "---------";
+            const idChanged = row.querySelector("input[id$=-id]")?.value;
 
-            questionnaireChanged = false;
-            $(row.find("input[name$=-questionnaires]")).each(function () {
-                if (this.checked)
+            var questionnaireChanged = false;
+            row.querySelectorAll("input[name$=-questionnaires]")?.forEach(el => {
+                if (el.checked)
                     questionnaireChanged = true;
             })
```

Comment:
Would
```suggestion
            const questionnaireChanged = row.querySelectorAll("input[name$=-questionnaires]:checked") !== null;
```
do the job?

Reply:
is `!== null` correct or do we need `.any()`? (possibly `?` before?)

Reply:
Ah, yeah, I may have tested locally with `querySelector` instead of `querySelectorAll`